analysis/prelim_strategy_analysis.md

```diff
@@ -5,43 +5,57 @@ author: David Grisham
 header-includes:
     - \usepackage{mathtools}
     - \DeclarePairedDelimiter\abs{\lvert}{\rvert}
+    - \newcommand{\Network}{\ensuremath{\mathcal{N}}}
     - \newcommand{\Nbhd}[1]{\mathcal{N}_{#1}}
 ...
 
-In this document, we analyze 3 strategies for a simple 2-player Bitswap
-infinitely repeated game.
+In this paper, we analyze 3 strategies for a simple 2-player Bitswap infinitely
+repeated game. We start by defining the system in the most general case, then do
+an analysis on a system subject to simplying constraints.
 
-**TODO**: finish this intro. Might want to mention the paragraph below, but have
-to be clear that the first section is more general than the specific case done
-in the analysis
+System
+======
 
-In a given round, a player's utility is dependent on the actions each of the
-players took in the previous round -- in other words, the debt ratio only
-considers the immediately preceding round, rather than all previous rounds.
+We have a network \Network of $\abs{\Network}$ users. The terms *users*, *peers*
+and *players* will be used somewhat interchangeably, depending on context. Each
+of the users has a neighborhood of peers, which is the set of users they are
+connected to. Each pair of peers plays an infinitely repeated Bitswap game. Each
+user's neighborhood is constant -- so any given pair of peers is connected for
+the entire repeated game. This means that the network topology is static as
+well.
+
+**TODO**: spacing after \Network\ within regular text is a bit too wide
 
 Actions and Utility Functions
-=============================
+-----------------------------
+
+**TODO**: check that all uses of $t$ from here down are consistent, did a lot of
+updates recently
+
+**TODO**: ensure lower bound for $t$ is always 0 (and not 1)
 
 A player has two possible actions: reciprocate ($R$) or defect ($D$). The
 utility function for player $i$ at time $t$ $u_i^t$:
 
 $$
-u_i^t = \sum_{j \in \Nbhd{i}} \delta_{a_j^{t}R} S_j(d_{ji}, \mathbf{d}_j^{-i}) B
+u_i^t = \sum_{j \in \Nbhd{i}} \delta_{a_j^{t}R}
+              S_j(d_{ji}^t, \mathbf{d}_j^{-i,t}) B
          - \delta_{a_i^t R} B \\
```

Comment:
Can you elaborate on what you mean by using `/` to add spaces? And regarding `\cdot`, are you saying that it might be useful to expand the series out to a few terms to make it more readable/understandable?

Reply:
if you use a forward slash with a space in math mode, it forces a space; makes it a bit easier to parse when reading.  When I first looked at it, it took looking at the definitions below to understand where the boundary between terms even was.
And you a good point; I take back what I said about factoring out the $B$.  Keep that as is. 👍

Reply:
Ah, I see, okay I'll play with the spacing to make it less cluttered. Good suggestion, thanks!

Reply:
Going to assume you meant `\` ;)

Reply:
Yeah, I really like the result of all of that but maintainability just dropped heavily (I did spacing around operators and such too, so if we ever want to change any of this from `\,` to `\ `, for example...). I'll work on a slightly maintainable solution to this, but in the meantime I think the pdf looks better.